plugins/linguist-backend/README.md

```diff
@@ -85,6 +85,14 @@ return createRouter({ schedule: schedule, age: { days: 30 } }, { ...env });
 
 With the `age` setup like this if the language breakdown is older than 15 days it will get regenerated. It's recommended that if you choose to use this configuration to set it to a large value - 30, 90, or 180 - as this data generally does not change drastically.
 
+## Offline
```

Comment:
Same, here, if you change this to use the full options then this would just be a pointer to the Linguist-JS documentation for all the options.